### DISCUSSION TITLE
How to setup passkey support

### DISCUSSION THREAD

**Discussion by PythonCoderAS** [goauthentik/authentik#7395] [Q&A] [answered]
A lot of sites have support for **passkeys**, which is similar to WebAuthN 2FA except it allows a user to sign in without the password, similar to a social login. However, I'm not sure how to set this up with Authentik.

**Comment by Rustymage** [comment, marked as answer]
https://www.youtube.com/watch?v=aEpT2fYGwLw

**Reply by tincanfury** [reply]
this link does not implement passkeys.

**Reply by Rustymage** [reply]
> this link does not implement passkeys.

No?

![image](https://github.com/user-attachments/assets/83a03cc0-6130-40fe-978a-bc4ec78f3964)
